frontend/src/employee-frontend/components/unit/unit-reservations/StaffAttendanceTable.tsx

```diff
@@ -328,8 +315,41 @@ const AttendanceRow = React.memo(function AttendanceRow({
       <StyledTd partialRow={false} rowIndex={rowIndex} rowSpan={1}>
         {editing ? (
           <EditStateIndicator
-            status={saveRequestStatus}
-            stopEditing={() => setEditing(false)}
+            status={Success.of()}
+            stopEditing={() => {
+              setEditing(false)
+              const savePromises = values.flatMap(({ date, timeRanges }) =>
```

Comment:
This could be just `values.map(...)` because you're returning a promise from the function in every case. Or you could replace `return Promise.resolve()` in the end with `return []` to take full advantage of the flatmap.

Reply:
Actually the map on the following row returns an array of promises, which most often contains only one promise, but may contain several in case the employee has several attendance markings for the same date.